controllers/account.go

```diff
@@ -291,20 +293,36 @@ func (c *ApiController) GetUserinfo() {
 	c.ServeJSON()
 }
 
-// GetHumanCheck ...
+// GetCaptcha ...
 // @Tag Login API
-// @Title GetHumancheck
-// @router /api/get-human-check [get]
-func (c *ApiController) GetHumanCheck() {
-	c.Data["json"] = HumanCheck{Type: "none"}
-
-	provider := object.GetDefaultHumanCheckProvider()
-	if provider == nil {
+// @Title GetCaptcha
+// @router /api/get-captcha [get]
+func (c *ApiController) GetCaptcha() {
+	applicationId := c.Input().Get("applicationId")
+	isPreview := c.Input().Get("isPreview")
+	captchaProvider := &object.Provider{}
+	if isPreview == "true" {
+		provider, err := object.GetPreviewCaptchaProvider(applicationId)
+		if provider == nil || err != nil {
+			c.ResponseError(err.Error())
+			return
+		}
+		captchaProvider = provider
+	} else {
+		provider, err := object.GetAppConfigCaptchaProvider(applicationId)
+		if provider == nil || err != nil {
```

Comment:
Don't add any additional but useless checks, mostly you just need to check `err != nil`